.eslintrc

```diff
@@ -30,7 +30,7 @@ rules:
   jsx-quotes: [2, prefer-double]
   keyword-spacing: 2
   key-spacing: 2
-  max-len: [2, 120, 4]
+  max-len: [2, 120, 4, {"ignoreComments": true}]
```

Comment:
I'm again this change.
It's the minimum value so that any line fit into the GitHub web view.